config-generators/cosmosdb_nosql-commands.txt

```diff
@@ -12,4 +12,6 @@ add Earth --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.earth" -
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:create" --fields.include "id" --fields.exclude "name"
 update Earth --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:read" --fields.include "id,type" --fields.exclude "name"
```

Comment:
can you try once with include `all` and exclude `one`, for example: --include "*" and --exclude "name"

Reply:
@abhishekkumams Added test for verifying this.